legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitLabProjectApi.java

```diff
@@ -85,6 +85,8 @@ public class GitLabProjectApi extends GitLabApiWithFileAccess implements Project
     private final ProjectStructureExtensionProvider projectStructureExtensionProvider;
     private final ProjectStructurePlatformExtensions projectStructurePlatformExtensions;
 
+    public static final String PROJECT_CONFIGURATION_WORKSPACE_ID_PREFIX = "ProjectConfiguration_";
```

Comment:
If possible, make this package private.

Reply:
This remains unresolved. Is there some reason this must be public? If not, please make it package private.

Reply:
Package private should be sufficient for visibility in GitLabProjectConfigurationAPI.